tests/test_op10.py

```diff
@@ -0,0 +1,106 @@
+import unittest
+from sklearn import datasets
+from sklearn import linear_model
+from sklearn.cluster import KMeans
+from sklearn.linear_model import LogisticRegression
+from sklearn.ensemble import VotingClassifier
+from sklearn.gaussian_process import GaussianProcessRegressor
+from sklearn.mixture import GaussianMixture, BayesianGaussianMixture
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import FloatTensorType
+from skl2onnx.common.data_types import onnx_built_with_ml
+
+
+class TestOp10(unittest.TestCase):
+    def _fit_model_binary_classification(self, model):
```

Comment:
These functions are defined in test_utils, you can just use that.
https://github.com/onnx/sklearn-onnx/blob/679bbdc0465ed86235e71a982b3e62edcbfb0e3d/tests/test_utils/tests_helper.py#L51

Reply:
fixed